windows-terminal-quake/Native/FocusTracker.cs

```diff
@@ -40,5 +42,49 @@ public static void FocusGained(Process process)
 				}
 			});
 		}
+
+		// Checks to see if the current window in focus should ignore hotkey
+		public static bool CheckFocusIgnoreHotKey()
+		{
```

Comment:
Perhaps it's a good idea to start with a check whether there are any processes configured that take precedence. The "Process.GetProcesses()"-call can be fairly expensive, so that would prevent a couple cpu cycles.

Reply:
I'm not going to lie :P I'm very new to user32/winapi calls. I'll take a look at what native functions I can use for this :)